extensions/omni/omni.c

```diff
@@ -482,7 +482,11 @@ static void register_hook(const omni_handle *handle, omni_hook *hook) {
   // Ensure we allocate in the top memory context as we need to keep these around forever
   MemoryContext oldcontext = MemoryContextSwitchTo(TopMemoryContext);
 
-  if (hook->wrap) {
+  if (hook->position == 0) {
```

Comment:
can use `omni_hook_position_default` instead of `0`

Reply:
Good point. Will make it much more readable.